python/lsst/sconsUtils/eupsForScons.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 try:
-    from eups import *
+    from eups import *  # noqa F403 F401 Importing everything into this namespace
```

Comment:
Please at least expand on the comment a bit. If I was coding it I'd use a list, so the linter could detect undefined symbols. But I think the current code is defensible, especially if the list of all needed eups symbols is very large.